pkgs/tools/backup/luckybackup/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchurl
+, pkgconfig, libtool
+, rsync, ssh, qmake
+}:
+
+with stdenv.lib;
+stdenv.mkDerivation rec {
+
+  pname = "luckybackup";
+  version = "0.5.0";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/luckybackup/${version}/source/${pname}-${version}.tar.gz";
+    sha256 = "0nwjsk1j33pm8882jbj8h6nxn6n5ab9dxqpqkay65pfbhcjay0g8";
+  };
+
+  buildInputs = [ rsync ssh ];
+
+  nativeBuildInputs = [ pkgconfig libtool qmake ];
+  
+  prePatch = ''
+    for File in luckybackup.pro menu/luckybackup-pkexec \
+        menu/luckybackup-su.desktop menu/luckybackup.desktop \
+        menu/net.luckybackup.su.policy src/functions.cpp \
+        src/global.cpp src/scheduleDialog.cpp; do
+      substituteInPlace $File --replace "/usr" "$out"
+    done
+  '';
+
+  enableParallelBuilding = true;
```

Comment:
This can be removed:

> Unless set to false, some build systems with good support for parallel building including cmake, meson, and qmake will set it to true.

Reply:
@aanderse OK. 

Something else?